### DISCUSSION TITLE
V2.76.1

### DISCUSSION THREAD

**Discussion by zhongyang219** [zhongyang219/MusicPlayer2#533] [Announcements]
紧急修正无法自动关联歌词文件的问题。
建议所有使用V2.76版本的用户都更新到此版本。

<hr /><em>This discussion was created from the release <a href='https://github.com/zhongyang219/MusicPlayer2/releases/tag/V2.76.1'>V2.76.1</a>.</em>

**Comment by CarterAli** [comment]
你好，我无法搜索到歌词，所有歌曲均显示【没有找到歌曲】不知是什么原因。

**Comment by xiaowu-li** [comment]
在WIN11上运行一段时间或者全屏后一段时间,电脑的任务栏会卡主无法操作,需要在任务管理器重启资源管理器

**Comment by cruih** [comment]
![2024-01-03_161821](https://github.com/zhongyang219/MusicPlayer2/assets/64239615/21728dff-42c3-4d1e-a7c9-376fd4bca24f)
![2024-01-03_162108](https://github.com/zhongyang219/MusicPlayer2/assets/64239615/310afab1-e9f6-44ae-bbb1-cf78d344430b)
win11，桌面歌词，移动到任务栏后，切换到其他程序，歌词就不前置显示了。

**Comment by xq270107266** [comment]
用了几十年千千静听 ，今天总算遇到大佬这款音乐播放软件了，很好！ 不喜欢在线听歌，总是喜欢下来本地播放

**Comment by loo123-cloud** [comment]
软件播放某些歌时一到30秒就自动播放其它歌，好奇怪呀🤣

**Reply by xiaowu-li** [reply]
这个情况是下载音乐平台的锅，因为有的平台不来会员只能预览30秒，所以下载下来就30秒

**Reply by loo123-cloud** [reply]
关键是我这首歌是首完整的歌啊，只要不播放“第30秒”就不会自动播放其它歌。所以才说奇怪啊！

**Comment by fackyourmother** [comment]
ffmpeg播放卡顿或者有杂音，还有莫名其妙的倍速播放，就是1秒钟播放完三分钟的歌，跟电脑性能有关系的吗

**Reply by loo123-cloud** [reply]
我刚才下了ffmpeg_core V1.0.0.1版本用ffmpeg模式播放音乐测试了一会，ogg,aac,mp3都可以正常播放啊，没有出现你说的这些问题。1秒钟播放完三分钟的歌这种情况应该是识别失败，可以尝试给音频转一遍码试试。

**Reply by fackyourmother** [reply]
那跟我格式是flac，dff之类，还有24bit，96khz有关系吗，用正常模式播就没问题，一开ffmpeg就概率性出现电音或者卡顿，不仅仅是接解码器和耳机时有问题，电脑扬声器外放也会有问题

**Reply by loo123-cloud** [reply]
应该是你的音频码流比较高，系统性能有点跟不上就会出现播放卡顿或者杂音，可以试着修改电源计划为卓越性能，应该就不会有杂音了，但会很耗电。

**Comment by haneko111** [comment]
你好，请问原来的界面2不能改回去吗，我想要有一个界面只显示两行歌词的，目前所有的界面都没有一个界面是显示两行歌词的。我用了这个软件很久，真的很喜欢，但是这个问题很影响我的使用……

**Reply by lrisora** [reply]
目前最新Action大概这个感觉，我也喜欢这个两行歌词
歌词是 单行/双行/多行 不仅取决于当前窗口大小下ui2.xml内lyrics元素的高度
当前歌词字号也会影响
你可以修改ui2.xml，使其能够在你设置的字号及窗口大小下表现为双行

![圖片](https://github.com/zhongyang219/MusicPlayer2/assets/43085437/6e539206-debc-4535-b300-f94039530216)

**Comment by pjeeeee** [comment]
大佬，请问下一个版本什么时候发版。现在遇到一个问题，媒体库，我是先把音乐的文件换了路径，想把所有曲目中的信息全部删除了，然后再重新添加，然后发现有些曲目删除不了，一直存在所的曲目的列表中。

**Reply by zhongyang219** [reply]
> 大佬，请问下一个版本什么时候发版。现在遇到一个问题，媒体库，我是先把音乐的文件换了路径，想把所有曲目中的信息全部删除了，然后再重新添加，然后发现有些曲目删除不了，一直存在所的曲目的列表中。

你可以试试在“设置”——“媒体库”——“清理媒体库”中点击“清理”按钮，把里面的勾全部勾上，点击确定。另外只要确保“设置”——“媒体库”中“启动时自动更新媒体库”的选项勾选上了，并且你音乐文件的目录添加到了媒体库目录中，那么程序启动时就会自动扫描媒体库目录里的音乐文件并添加到媒体库中来。

**Comment by sunnypizza43** [comment]
大佬，如何将正在播放曲目和播放记录同步到LastFM？

**Reply by sunnypizza43** [reply]
感谢大佬感谢，我已经看点啦，谢谢大佬开发，这是我最喜欢用的本地播放器啦！！！
